rules/msk_module_backend_test.go

```diff
@@ -0,0 +1,154 @@
+package rules
+
+import (
+	"path/filepath"
+	"testing"
+
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/stretchr/testify/require"
+	"github.com/terraform-linters/tflint-plugin-sdk/helper"
+)
+
+func Test_MskModuleBackend(t *testing.T) {
+	rule := NewMskModuleBackendRule()
+
+	tests := []struct {
+		Name     string
+		Files    map[string]string
+		WorkDir  string
+		Expected helper.Issues
+	}{
+		{
+			Name:    "backend doesn't have the team's suffix",
```

Comment:
nit/style: maybe this case should be near the end, so reading through the test cases it builds up: no config -> no s3 -> s3 missing key -> bad key -> good example